cloudinary_cli/modules/upload_dir.py

```diff
@@ -25,18 +25,27 @@
              "You can specify a whole path, for example folder1/folder2/folder3. "
              "Any folders that do not exist are automatically created.")
 @option("-p", "--preset", help="The upload preset to use.")
+@option("-e", "--exclude-dir-name", is_flag=True, default=False,
+        help="Exclude the directory name from the folder structure in Cloudinary. "
+             "This avoids creating the parent directory as a sub-folder of the --(f)older in Cloudinary. "
+             "When this option is used the contents of the directory is uploaded instead of the directory itself. ")
```

Comment:
```suggestion
             "When this option is used, the contents of the parent directory are uploaded instead of the parent directory itself and thus the name of the specified parent directory is not included in the pubic ID path of the uploaded assets.)
```

Reply:
@const-cloudinary - I'd like to suggest that we remove the description lines 29 & 30, and keep only this slightly expanded version of line 31.  I think the clarification that the option uploads the contents instead of the directory is the most clear explanation and with the additional phrase about it "thus" not including the parent directory name, I think we explain everything necessary in phrasing that works well for both fixed folder mode & dynamic folder mode.  

(Lines 29 & 30 get messy/confusing when trying to make the wording fit both modes, and don't add new info.  We'll include examples in the docs to provide additional clarity).